charts/authentik/templates/worker/deployment.yaml

```diff
@@ -44,6 +44,10 @@ spec:
       {{- end }}
       {{- if .Values.serviceAccount.create }}
       serviceAccountName: {{ include "authentik-remote-cluster.fullname" .Subcharts.serviceAccount }}
+      {{- else }}
+      {{- with .Values.server.serviceAccount }}
```

Comment:
```suggestion
      {{- with .Values.worker.serviceAccount }}
```

Reply:
Also, this will conflict with the service account we set just above. We probably should not set the above value if overridden in the values, and specify in the values that this will override the value set by `serviceAccount.create`.

Reply:
okay i switch it around - so that `serviceAccount.create` has only a effect, when `worker.serviceAccount` is not set.